### PR TITLE
Update version to 1.8.0-dev, update g word to 'generous'

### PR DESCRIPTION
**Pull request by murgatroid99**

### CODE REVIEW

File: BUILD [hunk 1]
```diff
@@ -34,11 +34,11 @@ load(
 )
 
 # This should be updated along with build.yaml
-g_stands_for = "gambit"
+g_stands_for = "generous"
 
 core_version = "4.0.0-dev"
```
Comment (nicolasnoble) [inline BUILD]: For consistency, could you update core to 5.0.0-dev please ?

Comment (murgatroid99) [reply BUILD]: I didn't change it in either file, and yet they have different numbers. Are you sure `5.0.0-dev` is the correct value?

**Comment by nicolasnoble** [comment]
Yeah, whomever changed it to 5.0.0-dev in build.yaml forgot to update BUILD.

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Pull request merged**